.github/workflows/validate.yaml

```diff
@@ -1,33 +1,23 @@
 name: Validate
 on:
-  push:
-    branches-ignore:
+  pull_request:
+    branches:
       - master
+    paths:
+      - 'src/**.md'
+
 jobs:
   front-matter:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
         uses: actions/checkout@v2
-      
-      - name: Get pull request information
-        uses: jwalton/gh-find-current-pr@6025da8f655f241d580c1b3a07f2f0a781349ed8 # v1.1.0
-        id: findPr
 
       - name: Install dependencies
-        if: success()
         run: pip3 install colored
-            
-      - name: Get list of changed files
-        id: fileList
-        if: success()
-        env:
-          URL: https://api.github.com/repos/${{ github.repository }}/pulls/${{ steps.findPr.outputs.pr }}/files
-        run: |
-          curl --fail -sS $URL | jq -r .[].filename | sort -u > files.txt
-          cat files.txt
```

Comment:
Can we keep this step, printing the list to stdout once?

Reply:
Oops, I had enabled automatic merge on approval.

The list is not printed in a separate step, but `tee` will print to stdout and write to file in a singe call. 

I can separate these if you want in a fresh PR